sequencer/src/options.rs

```diff
@@ -197,10 +199,18 @@ pub struct Options {
     /// The maximum number of bytes we will send in a single Libp2p gossip message
     #[clap(
         long,
-        env = "ESPRESSO_SEQUENCER_LIBP2P_MAX_TRANSMIT_SIZE",
+        env = "ESPRESSO_SEQUENCER_LIBP2P_MAX_GOSSIP_TRANSMIT_SIZE",
```

Comment:
Can you make sure all these libp2p vars we've added recently are in `public-env-vars.toml`? Might come in handy one day to look up a node's p2p config via it's HTTP server